com.microsoft.mrtk.input/Interactors/Poke/PokeInteractor.cs

```diff
@@ -23,50 +23,26 @@ public class PokeInteractor :
     {
         #region PokeInteractor
 
-        [Header("Poke interactor settings")]
-
-        [SerializeField]
-        [Tooltip("The XRNode on which this hand is located.")]
-        private XRNode handNode;
-
-        /// <summary>
-        /// The XRNode on which this hand is located.
-        /// </summary>
-        protected XRNode HandNode => handNode;
-
-        [Header("Pose settings")]
-
-        [SerializeField]
-        [Tooltip("Which specific hand joint does this interactor track?")]
-        private TrackedHandJoint joint;
-
-        /// <summary>
-        /// Cached reference to hands aggregator for efficient per-frame use.
-        /// </summary>
-        protected HandsAggregatorSubsystem HandsAggregator => handsAggregator ??= HandsUtils.GetSubsystem();
-        private HandsAggregatorSubsystem handsAggregator;
+        [SerializeReference]
+        [InterfaceSelector]
+        [Tooltip("The pose source representing the hand joint this interactor tracks")]
+        private IPoseSource jointPoseSource;
```

Comment:
nit: could we call this the "pokePoseSource"? It might not always have a hand joint driving it

Reply:
Done